src/style/separator/separator.tsx

```diff
@@ -7,7 +7,7 @@ interface SeparatorProps {
 
 export const Separator = ({ size, horizontal }: SeparatorProps) => {
   const dimension = horizontal ? 'width' : 'height';
-  const height = size === 'small' ? '16px' : size === 'large' ? '40px' : '8px' ;
-  
+  const height = size === 'small' ? '8px' : size === 'large' ? '16px' : '24px';
```

Comment:
a prop `large` nao deveria indicar a maior altura em pixels? Do jeito que o ternario foi definido, se size for `large`, o height recebe o valor de 16px.

```suggestion
  const height = size === 'small' ? '8px' : size === 'large' ? '24px' : '16px';
```

Reply:
verdade